packages/api-nest/src/library/library.service.ts

```
@@ -56,7 +55,10 @@ export class LibraryService {
    * @param id - Library item ID
    * @returns Library item or null if not found
    */
-  async findById(userId: string, id: string): Promise<LibraryItemEntity | null> {
+  async findById(
+    userId: string,
+    id: string,
```

ESLint

[error] Delete `,`

(prettier/prettier)